Makefile

```
@@ -197,14 +215,8 @@ build-reporting-operator: $(REPORTING_OPERATOR_BIN_DEPENDENCIES) $(GOFILES)
 	mkdir -p $(dir $(REPORTING_OPERATOR_BIN_OUT))
 	CGO_ENABLED=$(CGO_ENABLED) GOOS=$(GOOS) go build $(GO_BUILD_ARGS) -o $(REPORTING_OPERATOR_BIN_OUT) $(REPORTING_OPERATOR_PKG)
 
-metering-manifests:
-	export \
-		METERING_OPERATOR_IMAGE_REPO=$(METERING_OPERATOR_IMAGE_REPO) \
-		METERING_OPERATOR_IMAGE_TAG=$(METERING_OPERATOR_IMAGE_TAG); \
-	./hack/generate-metering-manifests.sh
-
 $(DEPLOY_METERING_BIN_OUT): $(GOFILES)
-	go build -o $(DEPLOY_METERING_BIN_OUT) $(GO_PKG)/cmd/deploy-metering
+	go build $(GO_BUILD_ARGS) -o $(DEPLOY_METERING_BIN_OUT) $(DEPLOY_METERING_PKG)
```

Couldn't this just be:
```suggestion
	go build $(GO_BUILD_ARGS) -o $(DEPLOY_METERING_BIN_OUT) ./...
```

---

We're only interested in building the binary for the deploy-metering CLI for this Makefile target. Building the `cmd/deploy-metering/main.go` package will take a fraction of the time it will take to build the entire project root.

---

ah, got it.